app/models/spreadsheets/primary_school_spreadsheet.rb

```diff
@@ -16,31 +16,29 @@ def import(file_path)
     end
   end
 
+  def process(row)
+    commune = get_commune(row)
+    return if commune.nil?
```

Comment:
@hengsokly please consider introducing exception handling and writing a log for a further investigation of why some records are not being saved.